packages/layouts/test/components/Layout.test.tsx

```diff
@@ -18,44 +12,44 @@ describe('<Layout />', () => {
     expect(wrapper.find('main').text()).toBe('Child');
   });
 
-  it('renders a sidebar', () => {
-    const sidebar = <SideBar accessibilityLabel="Test">Item</SideBar>;
-    const wrapper = shallow(<Layout sideBar={sidebar}>Child</Layout>).dive();
+  it('renders with props passed', () => {
+    const wrapper = shallow(
+      <Layout noBackground noPadding>
+        Child
+      </Layout>,
+    ).dive();
 
-    expect(wrapper.find('aside')).toHaveLength(1);
-    expect(wrapper.find('aside').contains(sidebar)).toBe(true);
+    expect(wrapper.find('main')).toHaveLength(1);
```

Comment:
doesn't look like this test is testing the props as passed. update the description?

Reply:
Technically it still is testing the additional props for code coverage, but we can't run expects on styles/class names.